frontend/TestRun/TestRun.svelte

```diff
@@ -133,7 +133,7 @@
     <div class="d-flex px-2 py-2 mb-1 border-bottom bg-white ">
         <div class="p-1">
             {#if testRun}
-                <a class="link-dark" href="/test/{testRun.test_id}/runs?additionalRuns[]={testRun.id}">
+                <a class="link-dark" href="/tests/{testInfo.test.plugin_name}/{testRun.id}">
```

Comment:
Why the `/{testInfo.test.plugin_name}` part is needed?
I guess it will be much more useful to have release name than just testing tool name.

For example:

`tests/releases/enterprise-2024.2/%test-id%` says clearly about the belonging of a test run to a release.
So it will be easy to dostinguish from the following:
`tests/staging/valerii/%test-id%`.

Reply:
> Why the `/{testInfo.test.plugin_name}` part is needed? I guess it will be much more useful to have release name than just testing tool name.
> 
> For example:
> 
> `tests/releases/enterprise-2024.2/%test-id%` says clearly about the belonging of a test run to a release.
> 
> So it will be easy to distinguish it from the following: `tests/staging/valerii/%test-id%`.
> 
> Which I personally find useful - we get info not even opening the link.

Testing tool is needed (or test-id, as it was previously) to determine which result table we query for those results, so it would have to stay. The release marker can be introduced, as an alternative URL, but release name is not something which can be guessed from testing frameworks themselves, aside from making them query the run. For argus, we can add another route which would act similarly to this one, like `/release/scylla-master/tests/scylla-cluster-tests/$id` or `/tests/scylla-cluster-tests/scylla-master/folder-name/test-name/$id`

Reply:
For the use case of getting from emails or elasticsearch data into Argus run

What's in this PR fits best, introducing more data into the url, complicates how we can build it where needed.

As suggested we can have more options for better sharing out of Argus, but we still need a way to get from test-id to Argus page